pkg/driver/driver.go

```diff
@@ -17,14 +17,11 @@ import (
 	"google.golang.org/grpc"
 )
 
-// CSIVersion is the version of the csi to set in the User-Agent header
-var CSIVersion = "dev"
```

Comment:
> [!NOTE]
Since this variable is not used, it has been merged into the Version variable :bulb: